cmd/version/version_test.go

```diff
@@ -5,25 +5,49 @@ import (
 	"testing"
 
 	"github.com/fatih/color"
+	gh "github.com/gojektech/proctor/cmd/version/github"
 	"github.com/gojektech/proctor/io"
+	"github.com/google/go-github/github"
 	"github.com/spf13/cobra"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestVersionCmdUsage(t *testing.T) {
-	versionCmd := NewCmd(&io.MockPrinter{})
+	githubClient := &gh.MockClient{}
+	versionCmd := NewCmd(&io.MockPrinter{}, githubClient)
 	assert.Equal(t, "version", versionCmd.Use)
 	assert.Equal(t, "Print version of Proctor command-line tool", versionCmd.Short)
 	assert.Equal(t, "Example: proctor version", versionCmd.Long)
 }
 
-func TestVersionCmd(t *testing.T) {
+func TestLatestVersionCmd(t *testing.T) {
 	mockPrinter := &io.MockPrinter{}
-	versionCmd := NewCmd(mockPrinter)
+	githubClient := &gh.MockClient{}
+	versionCmd := NewCmd(mockPrinter, githubClient)
+	version := "v0.6.0"
 
 	mockPrinter.On("Println", fmt.Sprintf("Proctor: A Developer Friendly Automation Orchestrator %s", ClientVersion), color.Reset).Once()
+	githubClient.On("LatestRelease", "gojektech", "proctor").Return(&github.RepositoryRelease{TagName: &version}, nil)
 
 	versionCmd.Run(&cobra.Command{}, []string{})
 
 	mockPrinter.AssertExpectations(t)
 }
+
+func TestOldVersionCmd(t *testing.T) {
+	mockPrinter := &io.MockPrinter{}
+	githubClient := &gh.MockClient{}
+	version := "v0.9.0"
```

Comment:
Can you make the version equal to "v1000.0.0" else the spec will fail when the client version is upgraded to "v0.9.0"